src/Drupal/DrupalAutoloader.php

```diff
@@ -127,7 +127,14 @@ public function register(Container $container): void
         $this->addThemeNamespaces();
         $this->registerPs4Namespaces($this->namespaces);
         $this->loadLegacyIncludes();
-        $checkDeprecatedHooksInApiFiles =  $drupalParams['bleedingEdge']['checkDeprecatedHooksInApiFiles'];
+
+        // Trigger deprecation error if checkDeprecatedHooksInApiFiles is enabled.
+        if ($drupalParams['bleedingEdge']['checkDeprecatedHooksInApiFiles']) {
+            trigger_error('The bleedingEdge.checkDeprecatedHooksInApiFiles parameter is deprecated and will be removed in a future release.', E_USER_DEPRECATED);
+        }
```

Comment:
ah this is how

Reply:
Just missing version, but that's OK I think?

Reply:
I've updated bleedingedge so when people copy that they won't get a deprecation notice.

Reply:
Might need an isset or something when we remove the setting? Or perhaps even now.

Reply:
We don't need the `isset`, we can remove this when it is removed from `extension.neon` and the schema.